client/src/components/hooks/useSquidList.jsx

```diff
@@ -0,0 +1,10 @@
+import { useQuery } from "react-query";
+
+import { ApiClient } from "../../backend/ApiClient";
+
+export const useSquidList = () =>
+  useQuery(["squids"], () => ApiClient.get("/squids").then((response) => response.data.squids), {
+    retry: false,
+    refetchOnWindowFocus: false,
```

Comment:
Let's actually disable this on the topmost level, in app.jsx -- https://react-query.tanstack.com/reference/QueryClient#_top